.changeset/cool-rockets-hug.md

```diff
@@ -0,0 +1,11 @@
+---
+"@kaizen/date-picker": patch
+---
+
+Fix a typescript error by bumping the minimum required @react-stately/datepicker package
+
+Version 3.3.0 of react-stately/datepicker generated a TS issue, which was handled in Kaizen with a @ts-expect-error comment. But newer releases (3.4.0 onwards) don't have the problem, and so no error is found.
+
+Because of a problem with Kaizen packaging accidentally shipping TS source files, consuming projects, which had different versions of this package, got TS errors saying that the "ts-expect-error" was unnecessary.
```

Comment:
Wouldn't say this is an accident, but more of an existing design 😅

Reply:
The `files: []` field in the package.json includes `!**/*.ts` which I assume was designed to exclude all TS files (and same for TSX too) but it doesn't work and probably has never worked.

At some point though someone thought "we shouldn't ship TS files" and tried to set it to that. But yeah its been shipping TS for a long time, probably forever

Reply:
Ohh that makes more sense... I'll try fix this instead of attempting bundling.